src/helpers.js

```diff
@@ -270,3 +270,26 @@ export function getChartCircle(chart, options) {
     height: options.radius * 2
   };
 }
+
+export function isBoundToPoint(options) {
+  return options && (options.xValue || options.yValue);
+}
+
+export function resolvePointPosition(chart, options) {
+  if (!isBoundToPoint(options)) {
+    const box = getChartRect(chart, options);
+    const point = getRectCenterPoint(box);
+    let radius = options.radius;
+    if (!radius || isNaN(radius)) {
+      radius = Math.min(box.width, box.height) / 2;
+      options.radius = radius;
+    }
+    return {
+      x: point.x,
```

Comment:
This makes me think if we should make all of the annotations be "boxes" in v2.
So each would have x,y,width,height and maybe centerX/centerY properties. 
Not sure what the benefirts would be other than consistency of the x/y between elements. Now its either top left corner or center depending of the type.

Reply:
When I was writing this PR, I see, for instance, box and line with x2 and y2 that could be useless, having x, y, width and height.
At the end of the day, all annotations are boxes, the line as well. I think it could make sense to normalize it. It could also help for the custom one. And anyway, every annotation could add own properties if needed, of top of them.

Reply:
Agreed. Care to write an issue about that for V2?

Reply:
@kurkle maybe this could be the content for new PR. What do you think?

Reply:
I think it would be a breaking change and we should do it for annotation plugin v2. So at this point I'd just write and issue about it so we remmeber to do it.